server/models.py

```diff
@@ -0,0 +1,18 @@
+from server import db
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
You can define here the type to be UUID in order to avoid assinging it in your controller logic.